profiles/views.py

```diff
@@ -62,28 +56,35 @@ class ProfileList(ListCreateAPIView):
      include_all: bool.
     """
     serializer_class = ProfileSerializer
-    permission_classes = (IsAuthenticatedOrReadOnly, )
+    permission_classes = [IsAuthenticatedOrReadOnly & SavedCompaniesListPermission]
     pagination_class = ForumPagination
 
     def get_queryset(self):
+        filters = FiltersQueryParamSerializer(data=self.request.query_params)
```

Comment:
Nope. It _must_ be implemented using this feature of DRF:

https://www.django-rest-framework.org/api-guide/filtering/#djangofilterbackend

Reply:
You can merge this PR if a task for fixing this is immediately created. Or fix it before merging.